POKEMON_OF_THE_DAY/PANDIUM.yaml

```diff
@@ -0,0 +1,41 @@
+version: 0.4
+base: node:16.14.2
+build: npm install --production && npm i --save-dev @types/node && npm run build
+run: node .
+configs:
+  schema:
+    properties:
+      pokemon_type:
+        type: string
+        $ref: '#/definitions/pokemon_types'
+      slack_user:
+        type: string
+        $ref: '#/definitions/slack_users'
+    required:
+      - pokemon_type
+      - slack_user
+    definitions:
+      slack_users:
+          type: number
+          oneOf:
+              -   title: Placeholder
+                  const: placeholder
+      pokemon_types:
+          enum:
+              -  placeholder  
+  uischema:
+    elements:
+    - type: Section
+      label: Configure Pokémon of the Day
+      hintText: Select which type of Pokémon you would like to learn about, and the Slack User who will receive the Pokémon of the day message.
```

Comment:
can you separate this hints and put them on the type and user selections?

Reply:
I originally had done that, but now I'm trying to follow the .yaml guidelines from our docs: https://app.gitbook.com/o/-MfEyg17bPwE_6Fs6Y-q/s/-MfJn-9R_dn6dvcGNcdk/build/anatomy-of-an-integration/pandium.yaml-spec/uischema#section
I errrm.... haven't followed these before, so this is a shift for me.